components/brave_vpn/brave_vpn_service_desktop.cc

```diff
@@ -82,10 +92,15 @@ std::string GetBraveVPNPaymentsEnv() {
 
 BraveVpnServiceDesktop::BraveVpnServiceDesktop(
     scoped_refptr<network::SharedURLLoaderFactory> url_loader_factory,
-    PrefService* prefs)
-    : BraveVpnService(url_loader_factory), prefs_(prefs) {
+    PrefService* prefs,
+    base::RepeatingCallback<mojo::PendingRemote<skus::mojom::SkusService>()>
+        skus_service_getter)
+    : BraveVpnService(url_loader_factory),
+      prefs_(prefs),
+      skus_service_getter_(skus_service_getter) {
   DCHECK(brave_vpn::IsBraveVPNEnabled());
-  DETACH_FROM_SEQUENCE(sequence_checker_);
+
+  EnsureMojoConnected();
```

Comment:
you don't need this here since you have it https://github.com/brave/brave-core/pull/10847/files#diff-2170e2d6e88ab6e0202eac0280482f5a45f468d0fcc8d9d4d48fc358812b4a0cR496

Reply:
Fixed in https://github.com/brave/brave-core/pull/10847/commits/1a9f60fc958fddf956f9bf2d69fac108b8e7fb09